docs/installation.rst

```diff
@@ -20,112 +20,137 @@ Installation
+
+2. Create the environment:
+
+   .. code-block:: bash
+
+       conda create --name tardis-{project_name} --file conda-{platform}.lock
+       
+   Replace ``{project_name}`` with a name for your TARDIS project.
 
-    $ conda create --name tardis --file conda-{platform}-64.lock
-    $ conda activate tardis
+3. Activate the environment:
 
-3. a. Developers should `fork the repository <https://github.com/tardis-sn/tardis/fork>`_, configure
+   .. code-block:: bash
+
+       conda activate tardis-{project_name}
+
+4. To install TARDIS, first execute these commands:
+
+   .. code-block:: bash
+
+      $ git clone git@github.com:tardis-sn/tardis.git
+      $ cd tardis
+      $ git remote add upstream git@github.com:tardis-sn/tardis.git
+      $ git fetch upstream
+      $ git checkout upstream/master
+    
+   The installation process differs for developers and non-developers:
+
+   a. Developers should `fork the repository <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo>`_ , configure
       GitHub to `work with SSH keys <https://docs.github.com/en/authentication/connecting-to-github-with-ssh>`_,
-      set up the `upstream remote <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork>`_,
-      and install the package in development mode.
+      set up the `upstream remote <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork>`_ and `origin` (pointing to your fork),
+      and install TARDIS in development mode.
 
-      ::
+      .. code-block:: bash
 
-        $ git clone git@github.com:<username>/tardis.git
-        $ cd tardis
-        $ git remote add upstream git@github.com:tardis-sn/tardis.git
-        $ git fetch upstream
-        $ git checkout upstream/master
         $ pip install -e .
 
       .. note::
 
         The complete developer guidelines can be found :ref:`here <developer_guidelines>`.
         
-    b. Non-developers can install from specific releases using pip-
+   b. Non-developers can install from specific releases using pip:
 
-      ::
+      .. code-block:: bash
 
         $ pip install git+https://github.com/tardis-sn/tardis.git@{tag}
 
-      For example- 
+      For example, to install the latest release:
 
-      ::
+      .. code-block:: bash
       
         $ pip install git+https://github.com/tardis-sn/tardis.git@release-latest
 
-      or trying the most recent, unreleased changes from upstream.
+      or to install the most recent, unreleased changes from upstream:
 
-      ::
+      .. code-block:: bash
 
         $ pip install git+https://github.com/tardis-sn/tardis.git@master
+        
+    .. note::
+      Running specific modules or tests will require additional optional dependencies. 
+      The `tardisbase` package is required for running TARDIS Regression Tests.
```

Comment:
This has produced nested `note` blocks, at least in the docs build linked in the PR.